Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -819,7 +819,18 @@
                         .withDataSourceType(Utility.KendoGrid.KendoGridColumnDataSourceType.Date)
                         .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Date)
                         .withStandardWidth(170)
-                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.RiskAssessmentDate)));
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.RiskAssessmentDate))
+                        .withInclusionCriterion(() => uiState.isBluePrintNodeAvailable(uiBluePrint.children.gdpr)))
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("PlannedRiskAssessmentDate")
+                        .withTitle("Dato for planlagt risikovurdering")
+                        .withId("PlannedRiskAssessmentDate")
+                        .withDataSourceType(Utility.KendoGrid.KendoGridColumnDataSourceType.Date)
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Date)
+                        .withStandardWidth(170)
```

Comment:
make sure the standard width is large enough to contain the name in one row:
![image](https://user-images.githubusercontent.com/13116889/207595409-12337e3f-fca2-40c4-a5d5-03ae8c24997f.png)